### PR TITLE
Fix #87

### LINKED ISSUE #87: Add abstraction level slide to writing methods slides
We should add an explaining slide to the writing methods slideset that shows that this is part of the abstraction levels.

### PR DESCRIPTION
**Pull request by nerdinand**
Add Levels of Abstraction to the "Writing Methods" slides

### CODE REVIEW

File: slides/writing-methods/index.html [hunk 1]
```diff
@@ -43,6 +43,40 @@ <h1>Ruby Monstas</h1>
...
 
+        <section>
+          <section>
+          <h2>Levels of Abstraction</h2>
+            <p class="fragment fade-in">In this course we will learn about 3 levels of abstraction:</p>
+            <table class="fragment fade-in">
+              <thead>
+                <tr><th>#</th><th>Concept</th><th>Deals with...</th></tr>
+              </thead>
+              <tr><td>1</td><td>Variables</td><td>Data</td></tr>
```
Comment (lxxxvi) [inline slides/writing-methods/index.html:54]: It's not important, but the `<tbody>...</tbody>` tag is missing around the table rows. 😉

File: slides/writing-methods/index.html [hunk 2]
```diff
@@ -43,6 +43,40 @@ <h1>Ruby Monstas</h1>
           </aside>
         </section>
 
+        <section>
+          <section>
+          <h2>Levels of Abstraction</h2>
+            <p class="fragment fade-in">In this course we will learn about 3 levels of abstraction:</p>
```
Comment (lxxxvi) [inline slides/writing-methods/index.html:49]:
```suggestion
            <p class="fragment">In this course we will learn about 3 levels of abstraction:</p>
```

I recently realized that `fade-in` is the default so I started to omit it.

Comment (lxxxvi) [reply slides/writing-methods/index.html:49]: You could remove `fade-in` in the entire document.

**Review by lxxxvi — APPROVED** [review]
I like it, thanks. I've added some unimportant comments.

File: slides/writing-methods/index.html (hunk 2 above)
Comment (nerdinand) [reply slides/writing-methods/index.html:49]: Didn't know that, thanks!

File: slides/writing-methods/index.html (hunk 1 above)
Comment (nerdinand) [reply slides/writing-methods/index.html:54]: Table syntax is the worst...

**Review by lxxxvi — APPROVED** [review]
🦒

**Comment by hharen** [comment]
As we discussed, adding the table also at the beginning and making clear we're just dealing with variables.

**Review by hharen — APPROVED** [review]
Well done ;)

**Pull request merged**